src/View.elm

```diff
@@ -63,10 +63,90 @@ timelineView statuses label =
         ]
 
 
+draftView : Model -> Html Msg
+draftView model =
+    let
+        hasSpoiler =
+            case model.draft.spoiler_text of
+                Nothing ->
+                    False
+
+                Just _ ->
+                    True
+    in
+        div [ class "col-md-3" ]
+            [ div [ class "panel panel-default" ]
+                [ div [ class "panel-heading" ] [ text "Post a message" ]
+                , div [ class "panel-body" ]
+                    [ Html.form [ class "form", onSubmit SubmitDraft ]
+                        [ div [ class "form-group checkbox" ]
+                            [ label []
+                                [ input
+                                    [ type_ "checkbox"
+                                    , onCheck <| DraftEvent << ToggleSpoiler
+                                    , checked hasSpoiler
+                                    ]
+                                    []
+                                , text " Add a spoiler"
+                                ]
+                            ]
+                        , if hasSpoiler then
+                            div [ class "form-group" ]
+                                [ label [ for "spoiler" ] [ text "Spoiler" ]
+                                , textarea
+                                    [ id "spoiler"
+                                    , class "form-control"
+                                    , rows 5
+                                    , placeholder "This text will always be visible."
```

Comment:
Let's be frank, I was highly confused with the official client and I still do. No mention of followup convos:

![screenshot at 20 18 40](https://cloud.githubusercontent.com/assets/41547/25246004/9b3237e0-2606-11e7-984d-87f952656cf9.png)